README.md

```diff
@@ -94,3 +93,98 @@ qiime emperor biplot \
+
+Another parameter worth thinking about is `--latent_dim`, which controls the number of dimensions used to approximate the conditional probability matrix.  This also specifies the dimensions of the microbe/metabolite embeddings `U.txt` and `V.txt`.  The most dimensions this has, the more accurate the embeddings can be -- but the higher the chance of overfitting there is.  The rule of thumb to follow is in order to fit these models, you need at least 10 times as many samples as there are latent dimensions (this is following a similar rule of thumb for fitting straight lines).  So if you have 100 samples, you should definitely not have a latent dimension of more than 10.  Furthermore, you can still overfit certain microbes and metabolites.  For example, you are fitting a model with those 100 samples and just 1 latent dimension, you can still easily overfit microbes and metabolites that appear in less than 10 samples -- so even fitting models with just 1 latent dimension will require some microbes and metabolites that appear in less than 10 samples to be filtered out.
+
+
+**Q** : What does a good model fit look like??
+
+**A** : Again the numbers vary greatly by dataset. But you want to see the both the `logloss` and `cv_rmse` curves decaying, and plateau as close to zero as possible.
+
+**Q** : How long should I expect this program to run?
+
+**A** : Both `epochs` and `batch-size` contribute to determining how long the algorithm will run, namely
+
+**Number of iterations = `epoch #` multiplied by the ( Total # of microbial reads / `batch-size` parameter)**
+
+This also depends on if your program will converge. The `learning-rate` specifies the resolution (smaller step size = smaller resolution, but may take longer to converge). You will need to consult with Tensorboard to make sure that your model fit is sane. See this paper for more details on gradient descent: https://arxiv.org/abs/1609.04747
+
+If you are running this on a CPU, 16 cores, a run that reaches convergence should take about 1 day.
+If you have a GPU - you maybe able to get this down to a few hours.  However, some finetuning of the `batch_size` parameter maybe required -- instead of having a small `batch_size` < 100, you'll want to bump up the `batch_size` to between 1000 and 10000 to fully leverage the speedups available on the GPU.
+
+Credits to Lisa Marotz ([@lisa55asil](https://github.com/lisa55asil)),  Yoshiki Vasquez-Baeza ([@ElDeveloper](https://github.com/ElDeveloper)) and Julia Gauglitz for their README contributions.
```

Comment:
Thanks for the inclusion here :) should be Vazquez with a z instead of Vasquez :P

Reply:
jesus 😆. fixed.